### ISSUE TITLE
sync is missing from the docs, and Unknown database problem

### ISSUE THREAD

**Issue by subatomicglue** [db-migrate/node-db-migrate#437]
Going here:
https://db-migrate.readthedocs.io/en/latest/
https://db-migrate.readthedocs.io/en/latest/Getting%20Started/the%20commands/

I do not see sync, which is in my --help

> `db-migrate --help
> Usage: db-migrate [up|down|reset|sync|create|db|seed|transition] [[dbname/]migrationName|all] [options]
> 
> db-migrate --version
> 0.10.0-beta.20
> `

Also, if I drop my database, then I can't figure out how to get it back, using db-migrate:
`> db-migrate db:drop app_test_database

> db-migrate db:create app_test_database
> [ERROR] Error: ER_BAD_DB_ERROR: Unknown database 'app_test_database'
> 
> db-migrate up
> [ERROR] Error: ER_BAD_DB_ERROR: Unknown database 'app_test_database'`
##

**Comment by wzrdtales** [comment]
@subatomicglue Sync was just introduced shortly, docs are about to being updated thanks for pointing this out though :)

With your second problem, this is a known problem. You have configured a database in your `database.json` which does not exist anymore. db-migrate currently always connects also for commands like db:create to this db in your config. Therefore you get the error thrown instead of the db created. This will need a few changes to all drivers and is already a working item.

So either you change your config, or you create the db by hand. 

See #338

**Comment by wzrdtales** [comment]
About sync: it is just the same as up and down, but it figures out by itself if the migration you define is on its way up or down.

**Comment by subatomicglue** [comment]
Makes sense.  Nice work on db-migrate, really loving it (coming from Flask SQL Alchemy previously).  Looking forward to the changes

**Comment by subatomicglue** [comment]
I'm sure you're aware, but I also noticed "seed" was missing from the docs as well:

`Usage: db-migrate [up|down|reset|sync|create|db|seed|transition] [[dbname/]migrationName|all] [options]`

https://db-migrate.readthedocs.io/en/latest/Getting%20Started/the%20commands/

**Comment by wzrdtales** [comment]
@subatomicglue Yeah, this needs to be cleaned up.

Seeds are postponed to a later point in time. They were originally part of the 0.10.x release, but as I want the current 0.10.x to get stable and switch afterwards to a release cycle with monthly releases I decided to postpone this instead and don't make the release dependent on that. Basically they are just what they sound like. A Data manipulation toolset, as migrations are really to moving a database from one state to another, but this can always include manipulating the data, which is not necessarily part of a migration.

I invested much time the last week into documentation and now take my time to finish those and back the new programable API with tests.
